topic_folders/communications/guides/submit-job-posting.md

```diff
@@ -0,0 +1,41 @@
+## Sharing Job Postings
+
+The Carpentries has two main mechanisms by which to share community job postings: One is within our newsletter, [Carpentries Clippings](https://carpentries.org/newsletter/) and the other is our [#jobs channel on Slack](https://join.slack.com/share/zt-jw6ndhkl-rKyqtKGiBZnVtojOUDtgLw)
```

Comment:
Discuss mailing list on TopicBox is also one of the key channels where opportunities are shared - so we have three main ones I suppose?